make_environment_ifort

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# -------------------------------------------------------------------------------------------------------------------------------------
+# GRASP2K ENVIRONMENT FLAGS - Intel ifort version
+# -------------------------------------------------------------------------------------------------------------------------------------
+#
+# Define the following global variables according to your environment and
+# source this script or add these definitions to your terminal configuration
+# file, eg. ~/.cshrc, ~/.bashrc or ~/.profile.
+#
+# Assumes that the Fortran compiler and OpenMPI/mpirun are added to your PATH
+# and that OpenMPI libraries are on LD_LIBRARY PATH
+#
+# -----------------------------------------------------------------------------------------------------------------
+# Set up main flags
+# -----------------------------------------------------------------------------------------------------------------
+export   FC=ifort                                              # Fortran compiler
+export   FC_FLAGS="-O2 -save "                                 # Serial code compiler flags
+export   FC_LD="-mkl=sequential"                               # Serial linker flags
+export   GRASP="${PWD}"                                        # Location of the 2018 root directory
```

Comment:
This is actually a problem if you `source` GRASP while not in the directory. I would switch this over to

```
export GRASP="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

from here: https://stackoverflow.com/a/246128/1601695

Reply:
Hmm yeah I thought about that too, the risk is that it makes the env file less transparent to the users - it is hard to directly understand what that line of code you suggest actually do without some serious stackoverflowing ;)  I suggest we keep it the way it is for now, and users who source from another dir then have to either include stepping to the grasp root dir in their source script, or modify the make env file.

Reply:
Morten is right.   I think this was present in GRASPdev  and I removed it because when sourced from another directory all kinds of things go wrong.   Otherwise there needs to be a check of some sort that the  CWD of the "source" command is the same as the PWD or something like that.